src/pip/_internal/commands/completion.py

```diff
@@ -89,7 +94,9 @@ def run(self, options, args):
                     prog=get_prog())
             )
             print(BASE_COMPLETION.format(script=script, shell=options.shell))
+            return SUCCESS
         else:
             sys.stderr.write(
                 'ERROR: You must pass {}\n' .format(' or '.join(shell_options))
             )
+            return ERROR
```

Comment:
This is a behavior change. This should return SUCCESS.

Why? Because that's the contract we've been following and it's likely that the auto-completion tools / logic is sensitive to this. If you're sure it's not sensitive, please elaborate on why you say so. :)

Reply:
Good catch, I didn't think of that. I also agree that such behaviour changes shouldn't be part of type annotation PRs,  hence changed this to `SUCCESS` in https://github.com/pypa/pip/pull/8051/commits/c83c2804caef4460094bad20b8ccf287dff58d36